cmd/exalysis/main_test.go

```diff
@@ -0,0 +1,45 @@
+package main
```

Comment:
Strictly speaking I could have added a test for this to one of the track tests, but I thought it might be useful to be able to test running the executable, too.